src/StreamJsonRpc/JsonMessageFormatter.cs

```diff
@@ -25,7 +25,7 @@ namespace StreamJsonRpc
     /// <summary>
     /// Uses Newtonsoft.Json serialization to serialize <see cref="JsonRpcMessage"/> as JSON (text).
```

Comment:
Do Dispose and Close methods do different things for this object?

Reply:
Which object?

Reply:
Close and Dispose should be equivalent on any I/O object that I'm familiar with.

Reply:
The comment was supposed to be on the line with var jsonReader = new JsonTextReader. I was wondering why you used finally instead of using

Reply:
I've switched to the using block syntax.